### DISCUSSION TITLE
Problem to display fraction

### DISCUSSION THREAD

**Discussion by templth** [remarkjs/remark#970] [Q&A] [answered]
Hello,

I'm trying to display math equations. It works fine with the following code:

```
<ReactMarkdown
  children={`
$$
\\sum_{n=0}^{N}\\frac{1}{2}
$$
`}
  remarkPlugins={[remarkMath]}
  rehypePlugins={[rehypeKatex]}
/>
};
```
I added the styles as well:

```
import 'katex/dist/katex.min.css'
```

However, the separator of fractions isn't displayed.

![image](https://user-images.githubusercontent.com/558344/159738019-be2ea032-58e4-4300-9c4a-1371416c3582.png)

I'm using the following versions of libraries:

```
    "react-katex": "^2.0.2",
    "react-markdown": "^8.0.1",
    "rehype-katex": "^6.0.2",
    "remark-math": "^5.1.1",
```

Thanks for your help

**Comment by wooorm** [comment, marked as answer]
Can’t reproduce: https://stackblitz.com/edit/github-zpydac?file=src/app.tsx.

**Reply by templth** [reply]
Was a bit strange. Perhaps a problem between versions of different libs. Now it works fine ;-) Thanks for your answer!
